invokeai/frontend/web/src/features/gallery/components/ImageGrid/GalleryImage.tsx

```diff
@@ -171,6 +171,7 @@ const GalleryImage = (props: HoverableImageProps) => {
                 borderTopEndRadius="base"
                 borderBottomStartRadius="base"
                 sx={badgeSx}
+                pointerEvents="none"
```

Comment:
This was `pointerEvents={isSelected ? 'auto': 'none'}`, but we never want the pointer events to be processed for the overlay. I changed it to just `"none"`.